### PR TITLE
feat(robot): add generators

### PR DESCRIPTION
**Pull request by davmarek**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature - new generator for `robot spec`

**What is the current behavior? (You can also link to an open issue here)**
Currently there is no generator for suggesting test cases

**What is the new behavior (if this is a feature change)?**
Now there is a generator for suggesting test cases from `.robot` files

### CODE REVIEW

File: src/robot.ts [hunk 1]
```diff
@@ -46,6 +46,38 @@ const variablesGenerator: Fig.Generator = {
   },
 };
 
+const testCasesGenerator: Fig.Generator = {
+  custom: async (tokens, executeShellCommand) => {
+    const out = await executeShellCommand(
+      'for i in $(find -E . -regex ".*.robot" -type f); do cat -s $i ; done'
+    );
```
Comment (mschrage) [inline src/robot.ts:53]: This doesn't need to be a custom generator. You can put this script in the `script` field and then use the `postProcess` function to handle the parsing logic.

Comment (davmarek) [reply src/robot.ts:53]: Will do

File: src/robot.ts [hunk 2]
```diff
@@ -46,6 +46,38 @@ const variablesGenerator: Fig.Generator = {
...
+        suggestions.push({
+          name: testCase,
+          description: "Test case",
+        });
+      }
+    }
+
+    return suggestions;
+  },
+};
```
[the remark this answers is not in the document]
Comment (davmarek) [reply src/robot.ts:79]: On it

File: src/robot.ts (hunk 1 above)
Comment (davmarek) [reply src/robot.ts]: I changed both `tagsGenerator` and `testCasesGenerator` to use the `script` field and `postProcess` function

File: src/robot.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (davmarek) [reply src/robot.ts]: I added comments to make it more understandable

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**